aries_cloudagent/connections/base_manager.py

```diff
@@ -294,6 +299,8 @@ async def resolve_invitation(
     def _extract_key_material_in_base58_format(method: VerificationMethod) -> str:
         if isinstance(method, Ed25519VerificationKey2018):
             return method.material
+        elif isinstance(method, Ed25519VerificationKey2020):
+            return bytes_to_b58(multibase.decode(method.material))
```

Comment:
I think this also needs an "unwrap" on the material to remove the `0xed` multicodec prefix added when using the `publicKeyMultibase`

Reply:
I added support for both (with and without multicodec). Do you think it should be like that or should we support multibase keys only with multicodec?

Reply:
Given the ambiguity, I think accepting either is probably a good choice for now at least.